src/types/functions.ts

```diff
@@ -11,6 +11,7 @@ export type ArgType =
   | "RANGE<NUMBER>"
   | "RANGE<DATE>"
   | "RANGE<STRING>"
+  | "RANGE<ANY>"
```

Comment:
"RANGE" do the same thing

Reply:
doesn't hurt to have an explicit alternative IMHO